### PR TITLE
Update for app config changes

### PR DESCRIPTION
**Pull request by chiatt**
Update for app config changes

### CODE REVIEW

**Comment by aarongundel** [comment]
@chiatt is this expected to be failing?

**Comment by jacobtylerwalls** [comment]
> It's complaining about missing pgtrigger.

Yeah, I would expect `arches_for_science` to be listed in pyproject.toml, but it's not.

I PR'd this to the dashboard toy project in https://github.com/chiatt/dashboard/pull/2/

File: pyproject.toml [hunk 1]
```diff
@@ -24,7 +24,8 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-    "arches>=7.6.0,<7.7.0",
+    "arches>=7.6.0a0,<7.7.0",
```
Comment (jacobtylerwalls) [inline pyproject.toml:27]: Guess we need a beta of arches or a pin to a github commit (for the time being)

**Comment by jacobtylerwalls** [comment]
Looks like a new package-lock.json is needed.

File: disco/settings.py [hunk 2]
```diff
@@ -441,60 +462,86 @@
...
         "ext": "txt",
-        "type": "text/plain",   
+        "type": "text/plain",
         "exclude": "",
     },
 ]
 
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
 FORMATS = [
```
Comment (aarongundel) [inline disco/settings.py:472]: This array isn't needed anymore per https://github.com/archesproject/arches-for-science/issues/1380

But shouldn't cause a problem.

**Review by aarongundel — APPROVED** [review]
minor nit - fix if desired.  LGTM.

**Pull request merged**